src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -35,6 +35,15 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
         return districtResult.partyResults;
     };
 
+    highLightWinnerRow = (winner: string): ComponentPropsGetter0 | ComponentPropsGetterR | undefined => {
```

Comment:
It might be possible to simply state the type of highLightWinnerRow explicitly here -- I mean, I see no way we could possibly return a ComponentPropsGetterR. Not important though.